app/middleware/skipLatLongLookup.js

```diff
@@ -0,0 +1,5 @@
+function skipLatLongLookup(res) {
```

Comment:
Rather than using this technique to conditionally execute route functions I was thinking it would be possible to use pattern matching within the route definitions to capture the query string params and route based on that. See https://expressjs.com/en/guide/routing.html#route-parameters

Reply:
I've looked into this, and there is no elegant way of routing on query strings other than using a regex.
The recent changes combine the place route into the results route uses the same conditionals. It might be worth getting together to look at it tomorrow, the current implementation isn't great - maybe we can find a way of improving the flow.